src/components/StaffList/index.jsx

```diff
@@ -10,10 +12,56 @@ class StaffList extends React.Component {
     super(props);
     this.state = {
       staffList: props.input.value || [], // requires staffList to be an array
+      suggestions: [], // the suggestions returned by the discovery service for autocomplete
```

Comment:
these comments seem a bit excessive to me, but maybe valuable to others? idk

Reply:
🤷‍♂️